Gemfile

```diff
@@ -136,7 +145,7 @@ group :development, :test do
   gem 'poltergeist'                     # for headless JS testing
   gem 'rspec-activemodel-mocks'
   gem 'rspec-rails' # unit testing framework
-  gem 'rubocop', '~> 0.60'
+  gem 'rubocop', '~> 0.70'
```

Comment:
Should do this separately on master and regenerate the config; so it doesn't ping this PR with issues that are existing already.